src/AutoPlay.ts

```diff
@@ -23,16 +25,19 @@ class AutoPlay implements Plugin {
   private _mouseEntered = false;
 
   public get duration() { return this._duration; }
+  public get animationDuration() { return this._animationDuration; }
   public get direction() { return this._direction; }
   public get stopOnHover() { return this._stopOnHover; }
 
   public set duration(val: number) { this._duration = val; }
+  public set animationDuration(val: number | undefined) { this._animationDuration = val; }
   public set direction(val: AutoPlayOptions["direction"]) { this._direction = val; }
   public set stopOnHover(val: boolean) { this._stopOnHover = val; }
 
   /**
    * @param {AutoPlayOptions} options Options for the AutoPlay instance.<ko>AutoPlay 옵션</ko>
    * @param {number} options.duration Time to wait before moving on to the next panel.<ko>다음 패널로 움직이기까지 대기 시간</ko>
+   * @param {number | undefined} options.animationDuration Duration of animation of moving to the next panel.<ko>패널이 움직이는 애니메이션의 지속 시간</ko>
```

Comment:
Please add an explanation that if it is undefined, the `duration` value of Flicking is used instead.
Both here & Flicking's demo page (plugin section).